versions/draft-3/SPEC.md

```diff
@@ -1515,6 +1522,130 @@ workflow wf {
 
 In this example, the fully-qualified names that would be exposed as workflow outputs would be `wf.task1.results`, `wf.altname.value`.
 
+## Struct Definition
+A struct is a C-like construct which enables the user to create new compound types that consist of previously existing types. Structs
+can then be used within a `Task` or `Workflow` definition as a declaration in place of any other normal types. The struct takes the place of the
+`Object` type in many circumstances and enables proper typing of its members.
+
+Structs are defined using the `struct` keyword and have a single section consiting of typed declarationns. They are evaluated first prior
+to workflows and tasks.
+```wdl
+struct name { ... }
+```
+
+### Struct Declarations
+The only contents of struct are a set of declarations. Declarations can be any primitive or compound type, as well as other structs, and are defined
+the same way as they are in any other section. The one caveat to this is that declarations within a struct do not allow an expression statement after
+the initial member declaration.
+
+for example the following is a valid struct definition
+```wdl
+struct name {
+    String myString
+    Int myInt
+}
+```
+Whereas the following is invalid
+
+```wdl
+struct invalid {
+    String myString = "Cannot do this"
+    Int myInt
+}
+```
+
+You can also use compound types within a struct to easily encapsulate them within a single object. For example
```

Comment:
reword to avoid the word "you"

Reply:
I think @katevoss might disagree with you! FWIW I personally much prefer examples in second-person rather than convoluted passive-voice sentences.

Reply:
@cjllanwarne is right! We encourage documentarians to address their readers directly with the second-person in a friendly yet clear manner. 

As our coffee mugs say,
>It's more important to be nearly right and understandable than to be academically accurate and unintelligible.

Reply:
So votes on language? Should we keep the voice referring to the user or move to a passive one

Reply:
I prefer the current form